### PR TITLE
Gitlab onboarding wizard

### PR DESCRIPTION
**Pull request by efelmandar**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**-->

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: 
https://jira-hq.paloaltonetworks.local/browse/CIAC-4202

## Description
Adding new content for XSIAM Gitlab package, for XSIAM Onboarding Wizard

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/GitLab/Playbooks/Gitlab_-_User_Permission_Changed_Alert.yml [hunk 1]
```diff
@@ -0,0 +1,660 @@
+id: Gitlab - User Permission Changed Alert
+version: -1
+name: Gitlab - User Permission Changed Alert
```
Comment (altmannyarden) [inline Packs/GitLab/Playbooks/Gitlab_-_User_Permission_Changed_Alert.yml:3]: I think we should rename and playbook and the correlation rule to Gitlab - Guest user permission change 
to indicate a guest user in involve.

File: Packs/GitLab/pack_metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "GitLab",
     "description": "Pack for handling gitlab operations",
     "support": "xsoar",
-    "currentVersion": "2.0.0",
+    "currentVersion": "2.1.0",
     "author": "Cortex XSOAR",
     "url": "https://www.paloaltonetworks.com/cortex",
     "email": "",
```
Comment (altmannyarden) [inline Packs/GitLab/pack_metadata.json:8]: Can also please fix the marketplaces in the pack metadata? 
should be for XSIAM only (marketplacev2)

**Comment by altmannyarden** [comment]
Approved from my side
@evisochek FYI

**Comment by ShirleyDenkberg** [comment]
@altmannyarden @evisochek Doc review completed.

File: Packs/GitLab/pack_metadata.json [hunk 3]
```diff
@@ -2,19 +2,20 @@
...
     ],
-    "tags": [],
+    "tags": [
+        "Data Source"
+    ],
     "useCases": [],
     "keywords": [],
     "marketplaces": [
-        "marketplacev2",
-        "xsoar"
```
Comment (yaakovpraisler) [inline Packs/GitLab/pack_metadata.json:18]: why did you remove it?

Comment (altmannyarden) [reply Packs/GitLab/pack_metadata.json:18]: because its only a pack with data source - not supported in XSOAR

Comment (altmannyarden) [reply Packs/GitLab/pack_metadata.json:18]: my mistake

**Review by evisochek — APPROVED** [review]
